### PR TITLE
fix(order): batch-quantity-bug

### PR DESCRIPTION
**Pull request by ammarhaiderak**

### CODE REVIEW

File: src/components/sales/Order.vue [hunk 1]
```diff
@@ -578,6 +579,8 @@ export default defineComponent({
     },
 
     productQuantityValidation: function () {
+
+      console.log('inside quantity validation function');
```
Comment (OmerAli277) [inline src/components/sales/Order.vue]: Remove console outputs

File: src/components/sales/Order.vue [hunk 2]
```diff
@@ -588,8 +591,11 @@ export default defineComponent({
           const value = parseFloat(this.product.quantity);
           if (isNaN(value)) {
             errorMessage = 'Only numbers are allowed';
-          } else {
-            if ( value > this.product.quantityUpperLimit) {
+          } else {            
+            
+            if ( value > this.selectedBatchQuantity) {
+              //console.log(`comparing ${value} > ${this.selectedBatchQuantity}`);
+              //console.log(this.productBatchSelect[parseInt(this.product.batch)-1]);
```
Comment (OmerAli277) [inline src/components/sales/Order.vue]: remove commented code

File: src/components/sales/Order.vue [hunk 3]
```diff
@@ -607,7 +626,10 @@ export default defineComponent({
       this.productDiscountValidation === null) {
         disable = false
       }
+
+      console.log('disable: '+disable);
```
Comment (OmerAli277) [inline src/components/sales/Order.vue]: remove

File: src/components/sales/Order.vue [hunk 4]
```diff
@@ -731,6 +753,11 @@ export default defineComponent({
     },
 
     addOrderItem: async function () {
+
+      // if(!this.addProductButton){
+      //   return;
+      // }
+
```
Comment (OmerAli277) [inline src/components/sales/Order.vue]: commented code again

**Review by OmerAli277 — CHANGES_REQUESTED** [review]

**Comment by ammarhaiderak** [comment]
change done please review.

**Review by OmerAli277 — APPROVED** [review]
Tested

**Pull request merged**